tree/dataframe/test/datasource_ntuple.cxx

```diff
@@ -19,6 +19,22 @@ using ROOT::Experimental::RNTupleModel;
 using ROOT::Experimental::RNTupleWriter;
 using ROOT::Experimental::Internal::RPageSource;
 
+namespace {
+
+class FileRAII {
```

Comment:
Can we reuse the `FileRAII` we have in ntuple_test.hxx?

Reply:
The RNTuple data source is in a different package (`tree/dataframe` vs. `tree/ntuple`) and I wanted to avoid dependencies across packages for this simple helper. One could factor it out into a general test helper but it also seems overkill give that, AFAIK, there is no common test helper infrastructure in place.